bluepysnap/circuit.py

```diff
@@ -49,6 +49,14 @@ def config(self):
         """Network config dictionary."""
         return self._config.to_dict()
 
+    def get_node_population_config(self, name):
+        """Get node population configuration."""
+        return self._config.populations["node"].get(name, {})
+
+    def get_edge_population_config(self, name):
+        """Get edge population configuration."""
+        return self._config.populations["edge"].get(name, {})
```

Comment:
I think these methods should raise a BluepySnapError if the population is not found.